test/index.js

```diff
@@ -0,0 +1,37 @@
+//
+// old-vs-new.js
+//
+
+//Imagine these are libs. Normally they would be require()'d.
+const oldVersion = (f, xs) => xs.map(f);
+const newVersion = (f, xs) => {
+  const len = xs.length; out = new Array(len);
+  for(let idx = 0; idx < len; idx++) out[idx] = f(xs[idx]);
+  return out;
+};
+
+const small = Array.from({length: 1}, (_, i) => i);
+const large = Array.from({length: 1000}, (_, i) => i);
+
+//Normally we wouldn't assign this to a var, but just export it.
+const benchmark = module.exports = require('..')(oldVersion, newVersion, {
+  'map/small': map => map(x => x + 1, small),
+  'map/large': map => map(x => x + 1, large)
+});
+
+
+//
+// index.js
+//
+
+//Here we would map command line arguments to benchmarking options. We would
+//also determine which benchmarks to require(). In this case I've hard-coded
+//the old-vs-new.js file, but we might just as well run a ramda-vs-sanctuary.js
```

Comment:
Nit: I like to include two spaces after `//`. This makes the text align with `//.`-style comments (with one space).